src/main/java/org/jenkinsci/plugins/electricflow/ElectricFlowPublishApplication.java

```diff
@@ -78,7 +78,7 @@ public ElectricFlowPublishApplication(String configuration, String filePath) {
   // ~ Methods ----------------------------------------------------------------
 
   public static FilePath makeApplicationArchive(
-      Run<?, ?> build, TaskListener listener, FilePath workspace, String filePath)
```

Comment:
@justnoxx , not sure why this had been added previously in the branch. So I left all this things as it was in the branch - Run<?, ?>